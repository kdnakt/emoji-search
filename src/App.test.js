import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';

import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const app = renderer.create(<App />);
  expect(app.toJSON()).toMatchSnapshot();
});
