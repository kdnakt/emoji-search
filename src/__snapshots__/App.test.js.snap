// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <header
    className="component-header"
  >
    <img
      height="32"
      role="presentation"
      src="//cdn.jsdelivr.net/emojione/assets/png/1f638.png"
      width="32"
    />
    Emoji Search
    <img
      height="32"
      role="presentation"
      src="//cdn.jsdelivr.net/emojione/assets/png/1f63a.png"
      width="32"
    />
  </header>
  <div
    className="component-search-input"
  >
    <div>
      <input
        onChange={[Function]}
      />
    </div>
  </div>
  <div
    className="component-emoji-results"
  >
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="100"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f4af.png"
      />
      <span
        className="title"
      >
        100
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="1234"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f522.png"
      />
      <span
        className="title"
      >
        1234
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Grinning"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f600.png"
      />
      <span
        className="title"
      >
        Grinning
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Grimacing"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f62c.png"
      />
      <span
        className="title"
      >
        Grimacing
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Grin"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f601.png"
      />
      <span
        className="title"
      >
        Grin
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Joy"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f602.png"
      />
      <span
        className="title"
      >
        Joy
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Smiley"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f603.png"
      />
      <span
        className="title"
      >
        Smiley
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Smile"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f604.png"
      />
      <span
        className="title"
      >
        Smile
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Sweat Smile"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f605.png"
      />
      <span
        className="title"
      >
        Sweat Smile
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Laughing"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f606.png"
      />
      <span
        className="title"
      >
        Laughing
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Innocent"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f607.png"
      />
      <span
        className="title"
      >
        Innocent
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Wink"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f609.png"
      />
      <span
        className="title"
      >
        Wink
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Blush"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f60a.png"
      />
      <span
        className="title"
      >
        Blush
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Slight Smile"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f642.png"
      />
      <span
        className="title"
      >
        Slight Smile
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Upside Down"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f643.png"
      />
      <span
        className="title"
      >
        Upside Down
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Relaxed"
        src="//cdn.jsdelivr.net/emojione/assets/png/263a.png"
      />
      <span
        className="title"
      >
        Relaxed
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Yum"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f60b.png"
      />
      <span
        className="title"
      >
        Yum
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Relieved"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f60c.png"
      />
      <span
        className="title"
      >
        Relieved
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Heart Eyes"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f60d.png"
      />
      <span
        className="title"
      >
        Heart Eyes
      </span>
    </div>
    <div
      className="component-emoji-result-row"
    >
      <img
        alt="Kissing Heart"
        src="//cdn.jsdelivr.net/emojione/assets/png/1f618.png"
      />
      <span
        className="title"
      >
        Kissing Heart
      </span>
    </div>
  </div>
</div>
`;
